docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/module_operations.md

```diff
@@ -37,7 +37,7 @@ direct_product(M::ModuleFP{T}...; task::Symbol = :prod) where T
 ## Truncation
 
 ```@docs
-truncate(M::ModuleFP, g::FinGenAbGroupElem, task::Symbol = :with_morphism)
+truncate(M::ModuleFP, g::FinGenAbGroupElem, task::Symbol=:with_morphism)
 ```
 
```

Comment:
Thr generic idea is/was to abolish `task` everywhere....

Reply:
That's what I said. See above.